Tutorial_5_BonusNotebook.ipynb

```diff
@@ -0,0 +1,756 @@
+    "`# (for later use for merging readcounts with gene names in R code).`\n",
+    "`esearch -db assembly -query GCF_000002765.6 | efetch -format docsum \\`  \n",
+    "`| xtract -pattern DocumentSummary -element FtpPath_RefSeq \\`  \n",
+    "`| awk -F\"/\" '{print \"curl -o data/reference/\"$NF\"_feature_table.txt.gz \" $0\"/\"$NF\"_feature_table.txt.gz\"}' \\`  \n",
+    "`| bash`  \n",
+    "\n",
+    "\n",
+    "`#unzip the compresseed fasta files`  \n",
+    "`gzip -d data/reference/*.gz --force`  \n",
+    "   \n",
+    "  \n",
+    "</details>\n",
+    "\n",
+    "\n",
+    "\n",
+    "\n",
+    "\n",
+    "\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now create a transcriptome reference file from the files we downloaded using the `gffread` command."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "<YOUR COMMAND HERE>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<details>\n",
+    "  <summary>Click for help</summary>\n",
+    "   \n",
+    "\n",
+    "`gffread -w data/reference/GCF_000002765.6_transcriptome_reference.fa -g data/reference/GCF_000002765.6_GCA_000002765_genomic.fna`   `data/reference/GCF_000002765.6_GCA_000002765_genomic.gff`  \n",
+    "\n",
+    "   \n",
+    "  \n",
+    "</details>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "It is also recommended to include the full genome at the end of the transcriptome reference file, for the purpose of performing a 'decoy-aware' mapping, more information about which can be found in the Salmon documentation. Note that a key difference between this and the first tutorial is that since we are using data from a eukaryotic organism, the genome comes in multiple chromosomes. That means that we will actually be pasting 14 sequences (_P. falciparum_ has 14 chromosomes) to the transcriptome file and writing 14 lines to the decoy file.\n",
+    "\n",
+    "There are a couple ways you could do this. One easy whay would be to simply copy and past the chromosome names into the decoys.txt file, but this does scale well if you end up using an organism with many chromosomes or if you make a user error. A better way is to programmatically extract the chromosome names from the genome fasta file. Try combining some `grep` and `sed` commands to pull the ID lines from the fasta file and extract the chromosome name from it. If you get stuck, we have one option in the hint box below."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<details>\n",
+    "  <summary>Click for help</summary>\n",
+    "   \n",
+    "\n",
+    "`# First use cat to combine the transcriptome and genome fasta files into a transcriptome_reference_w_decoy fasta file`  \n",
```

Comment:
missing code block for user to use.

Reply:
Fixed. I made the comments just regular markdown to help with readability and copy/paste a little easier.